xml/cap_admin_minibroker.xml

```diff
@@ -0,0 +1,291 @@
+ <info>
+  <title>Minibroker</title>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:bugtracker></dm:bugtracker>
+   <dm:translation>yes</dm:translation>
+  </dm:docmanager>
+ </info>
+ <para>
+  <link xlink:href="https://github.com/SUSE/minibroker">Minibroker</link> is an
+  <link xlink:href="https://www.openservicebrokerapi.org/">OSBAPI compliant
+  broker</link> created by members of the
+  <link xlink:href="https://github.com/osbkit">Microsoft Azure team</link>. It
+  provides a simple method to provision service brokers on Kubernetes clusters.
+ </para>
+ <sect1 xml:id="sec.cap.minibroker-deployment">
+  <title>Deploy Minibroker</title>
+
+  <procedure>
+   <step>
+    <para>
+     Minibroker is deployed using a &helm; chart. Ensure your SUSE &helm; chart
+     repository contains the most recent Minibroker chart:
+    </para>
+<screen>&prompt.user;helm repo update
+Hang tight while we grab the latest from your chart repositories...
+...Skip local chart repository
+...Successfully got an update from the "stable" chart repository
+...Successfully got an update from the "suse" chart repository
+Update Complete. ⎈ Happy Helming!⎈
+
+&prompt.user;helm search suse
+NAME                        	VERSION	DESCRIPTION                                       
+...
+suse/minibroker             	0.2.0  	A minibroker for your minikube                    
+...
+</screen>
+   </step>
+   <step>
+    <para>
+     Use &helm; to deploy Minibroker:
+    </para>
+<screen>&prompt.user;helm install suse/minibroker --name minibroker --namespace minibroker \
+        --set "helmRepoUrl=https://minibroker-helm-charts.s3.amazonaws.com/kubernetes-charts/" \
+        --set "deployServiceCatalog=false" \
+        --set "defaultNamespace=minibroker" \
+        --set "image=splatform/minibroker:latest"
+</screen>
+    <para>
+     The <literal>helmRepoUrl</literal> parameter points to a &helm; repository
+     containing
+     <link xlink:href="https://minibroker-helm-charts.s3.amazonaws.com/kubernetes-charts/index.yaml">charts
+     for the available services</link>.
+    </para>
+    <para>
+     The repository currently contains charts for the following services:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       <link xlink:href="https://mariadb.org">MariaDB</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="https://www.mongodb.com/">MongoDB</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="https://www.postgresql.org/">PostgreSQL</link>
+      </para>
+     </listitem>
+     <listitem>
+      <para>
+       <link xlink:href="http://redis.io">Redis</link>
+      </para>
+     </listitem>
+    </itemizedlist>
+    <warning>
+     <title>Do not use Minibroker services on production systems</title>
+     <para>
+      The services used by Minibroker are not production grade and are not recommended for use with production systems.
+     </para>
```

Comment:
see previous comment

Reply:
Moved to suggested spot in https://github.com/SUSE/doc-cap/pull/285/commits/61faf662457f2be81a9b0a10e7df009ceca79c68